wallet/src/database/dao/WalletDAO.ts

```diff
@@ -0,0 +1,39 @@
+import { ModelCtor, Sequelize } from "sequelize";
+import { CreateWalletResponse } from "../../server/protos/schema_pb";
+import { WalletModel } from "../model";
+
+export class WalletDAO {
+  private driver: Sequelize;
+  private wallet: ModelCtor<WalletModel>;
+
+  constructor(driver: Sequelize) {
+    this.driver = driver;
+    this.wallet = driver.model(WalletModel.tableName);
+  }
+
+  async createWallet({
+    telegram_from_user_id,
```

Comment:
The params should only be: `rapyd_e_wallet_address` and `user_id`. So that the wallet address remains linked to a user id.